src/OFS/zpt/main.zpt

```diff
@@ -3,290 +3,235 @@
 <tal:tabs replace="structure here/manage_tabs" />
 
 <main class="container-fluid">
-	<form name="objectItems" method="post"
```

Comment:
There are too many changes in this file, I am not able to check whether the code has not changed.

Reply:
I did a beautify of the file and arranged the attributes to a readable form.

Might not be easy to automatically compare.